pipelines/sync-hac-frontend-configs/tasks.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     This task updates redhat-appstudio-qe/frontend-configs with 
     content from internal gitlab repo insights-platform/frontend-configs.
   steps:
-    - image: quay.io/devfile/base-developer-image:latest
+    - image: quay.io/devfile/base-developer-image@sha256:latest
```

Comment:
Can you please revert changes in pipelines directory? We need to keep files in this directory as they are - mostly these pipelines have nothing to do with e2e-framework. In this this case the change will probably not work. Thank you!

Reply:
Thanks! My mistake. Fixed!